docs/md/iofogctl_delete_controller.md

```diff
@@ -25,8 +25,10 @@ iofogctl delete controller NAME
 ### Options inherited from parent commands
 
 ```
+      --detached           Use/Show detached resources
       --http-verbose       Toggle for displaying verbose output of API client
   -n, --namespace string   Namespace to execute respective command within (default "default")
+      --soft               Don't delete ioFog Stack from remote host
```

Comment:
Instead of referring to stack on host, can we use "uninstall Controller from host"?